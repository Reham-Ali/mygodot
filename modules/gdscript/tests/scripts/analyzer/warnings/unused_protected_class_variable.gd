var _a
@warning_ignore("unused_protected_class_variable")
var _b
@warning_ignore("unused_protected_class_variable") var _c
var _d

func test():
	pass
