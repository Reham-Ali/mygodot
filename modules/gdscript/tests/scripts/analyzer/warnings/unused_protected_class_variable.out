GDTEST_OK
>> WARNING
>> Line: 1
>> UNUSED_PROTECTED_CLASS_VARIABLE
>> The class variable "_a" that is protected is declared but never used in the class.
>> WARNING
>> Line: 5
>> UNUSED_PROTECTED_CLASS_VARIABLE
>> The class variable "_d" that is protected is declared but never used in the class.
